Firmware/Marlin_main.cpp

```diff
@@ -9474,15 +9475,16 @@ void restore_print_from_ram_and_continue(float e_move)
 //	for (int axis = X_AXIS; axis <= E_AXIS; axis++)
 //	    current_position[axis] = st_get_position_mm(axis);
 	active_extruder = saved_active_extruder; //restore active_extruder
-	if (saved_extruder_temperature) {
+	fanSpeed = saved_fanSpeed;
+	if (degTargetHotend(saved_active_extruder) != saved_extruder_temperature)
```

Comment:
Is it necessary to have the saved temperature and new target one equal? Wouldn't some hysteresis of e.g. 3°C help prevent meaningless waiting for the extruder to reach exact temperature?